types/test/renameKeyWith.ts

```diff
@@ -0,0 +1,6 @@
+import * as RA from 'ramda-adjunct';
+
+RA.renameKeyWith((str) => 'a' + str, 'A', { A: 1 }); // $ExpectType object
+RA.renameKeyWith((str) => 'a' + str)('A', { A: 1 }); // $ExpectType object
+RA.renameKeyWith((str) => 'a' + str)('A')({ A: 1 }); // $ExpectType object
+RA.renameKeyWith((str) => 'a' + str, 'A')({ A: 1 }); // $ExpectType object
```

Comment:
Adding some `$ExpectError` tests would be nice

Reply:
Will do